Intotech.GhostRider/Intotech.GhostRider/GeneratorRealization.cs

```diff
@@ -159,6 +160,23 @@ public virtual bool DtoLogicRender(string mainFolderPath, string outputDirectory
                 return true;
             }
         }
+
+        public virtual bool MDManRender(string folderPath, string baseClass)
+        {
+            try
+            {
+                ModelDerivanceManipulator mdm = new ModelDerivanceManipulator();
+
+                mdm.AddDerivanceToModels(folderPath, baseClass);
+
+                return true;
+            }
+            catch
```

Comment:
this will hide any error that there possibly can be, we have to at least log the error somewhere